src/config/MetaModules.json

```diff
@@ -0,0 +1,10 @@
+[
+    {
+        "name": "example",
+        "description": "An example meta module",
+        "steps":[
```

Comment:
So while this is cool, I wonder if we can specify meta modules in a serialized string that is readable as part of a url request. Maybe we need both options so like this OR `?steps=invert[settings...],green-channel, threshold [50]` just for example.

Reply:
@jywarren you are talking about the browser, right? Actually we have a separate issue to do this for the browser and it's a part of summer proposal as well, if you say i can start working on this right now?

Reply:
Actually, the ability to record both steps and settings in a string could be useful for browser work, for saving sequences in `localStorage`, in a key/value database on server or client side, and for REST based processing, so I think it's worth considering for all those reasons.

Great, though -- so for this PR, so as not to create an incompatibility, we should either think about a subset of the full syntax that will remain compatible after the settings-in-a-string feature, so while here we might just enable settings steps in JSON, we should expect to be able to detect a string submission and switch into a different parsing mode which can read in the settings. I think that means we are OK here - but maybe we should leave an inline comment saying we'll soon accept compact string format like `?steps=invert(settings...),green-channel,threshold(50)` 

Perhaps using parentheses will make it more like a function call... but we don't have to make a final decision just yet.

Reply:
See #127 for current planning on compact settings options in URLs